webapp/channels/src/packages/mattermost-redux/src/reducers/entities/scheduled_posts.ts

```diff
@@ -82,39 +106,59 @@ function errorsByTeamId(state: ScheduledPostsState['errorsByTeamId'] = {}, actio
         Object.keys(scheduledPostsByTeamId).forEach((teamId: string) => {
             if (scheduledPostsByTeamId.hasOwnProperty(teamId)) {
                 const teamScheduledPosts = scheduledPostsByTeamId[teamId] as ScheduledPost[];
-                const updatedHasError = teamScheduledPosts.some((scheduledPost) => scheduledPost.error_code);
+                const scheduledPostIDsWithError = teamScheduledPosts.
+                    filter((scheduledPost) => scheduledPost.error_code).
+                    map((scheduledPost) => scheduledPost.id);
 
-                if (state[teamId] !== updatedHasError) {
+                if (state[teamId] !== scheduledPostIDsWithError) {
```

Comment:
I don't think this will ever be true because `scheduledPostIDsWithError` will always be a new array

Reply:
You're right, I missed it when converting from it from bool to array. Fixed it now.